### PR TITLE
refactor(bridge): polish codes ⭐ 

### LINKED ISSUE #34: fix(bridge): correct slack bot libplanet-explorer url resolvement
Originally, it expects `https://explorer.libplanet.io/9c-internal/transaction/?3409cdbaa24ec6f7c8d2c0f636325a2b2e9611e5e6df5c593cfcd299860d8043` but `https://explorer.libplanet.io/transaction/?3409cdbaa24ec6f7c8d2c0f636325a2b2e9611e5e6df5c593cfcd299860d8043`. It'll make users confused.

### PR DESCRIPTION
**Pull request by moreal**
This resolves #34.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       working-directory: ./bridge
     - run: bash <(curl -s https://codecov.io/bash)
       name: codecov
-      working-directory: ./bridge
+      working-directory: ./bridge/test
```
Comment (longfin) [inline .github/workflows/build.yaml:22]: Is it temporary?

Comment (moreal) [reply .github/workflows/build.yaml:22]: No, `jest` created coverage result at `/bridge/test/coverage` and I changed the `working-directory` to match.

Comment (moreal) [reply .github/workflows/build.yaml:22]: But I found the configuration to move the directory's path so I will revert it.

**Comment by moreal** [comment]
PTAL @longfin

**Review by longfin — APPROVED** [review]

**Pull request merged**